homeassistant/components/tessie/entity.py

```diff
@@ -52,7 +52,7 @@ def get(self, key: str | None = None, default: Any | None = None) -> Any:
         return self.coordinator.data.get(key or self.key, default)
 
     async def run(
-        self, func: Callable[..., Awaitable[dict[str, bool | str]]], **kargs: Any
+        self, func: Callable[..., Awaitable[dict[str, Any]]], **kargs: Any
```

Comment:
Why is this change required?

Reply:
Translation key would not accept there being the chance the return value was bool.